content/birds/miriam.md

```diff
@@ -25,27 +25,27 @@ social:
   stackoverflow: 1930386
 summary: |
   Miriam is a co-founder and front-end architect --
-  helping oversee the user-experience
+  overseeing the user-experience
```

Comment:
@mirisuzanne I made this change so that your and my bios don't both start with `___ is a ___ -- helping ___`. 😄

Reply:
ha, fine - we can be _slightly_ different people